### DISCUSSION TITLE
Make your first PR! ~ A beginner-friendly repository made specifically for open source beginners. Add your profile, a blog, or any program under any language or update the existing one. Just make sure to add the file under the correct directory. Happy hacking!

### DISCUSSION THREAD

**Discussion by DHEERAJHARODE** [DHEERAJHARODE/Hacktoberfest2025-Open-source-#856] [Q&A]
Contribute your valuable codes here.

<hr /><em>This discussion was created from the release <a href='https://github.com/1dharode/Hacktoberfest2022/releases/tag/Hacktoberfest2022'>Make your first PR! ~ A beginner-friendly repository made specifically for open source beginners. Add your profile, a blog, or any program under any language or update the existing one. Just make sure to add the file under the correct directory. Happy hacking!</a>.</em>

**Comment by sumitawate2312** [comment]
@DHEERAJHARODE  Can you please accept my pull request , Have done all the pre-requisites
